packages/ckeditor5-dev-changelog/src/constants.ts

```diff
@@ -31,3 +31,12 @@ export const ISSUE_SLUG_PATTERN = /^(?<owner>[a-z0-9.-]+)\/(?<repository>[a-z0-9
 export const ISSUE_PATTERN = /^\d+$/;
 export const ISSUE_URL_PATTERN =
 	/^(?<base>https:\/\/github\.com)\/(?<owner>[a-z0-9.-]+)\/(?<repository>[a-z0-9.-]+)\/issues\/(?<number>\d+)$/;
+
+export const TYPES = [
+	{ name: 'Feature' },
+	{ name: 'Other' },
+	{ name: 'Fix', aliases: [ 'Fixes' ] },
+	{ name: 'Major', aliases: [ 'Major breaking change' ] },
```

Comment:
I don't know if displaying `Major` is the best solution. `Major` is not a `type`, and we display it in the logs:
![image](https://github.com/user-attachments/assets/5c94bb93-7844-4a59-9f77-328bbe9aefc3)

Reply:
Previously it looked like this. We were able to provide a `type`:
![image](https://github.com/user-attachments/assets/a77f9cc7-e348-4d0a-bebe-afd8b5a141b9)